tests/test.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import pycurl
+import io
+import json
+
+
+def test_response():
+    c = pycurl.Curl()
+    b = io.BytesIO()
+    c.setopt(pycurl.URL, 'http://localhost:5000/model/predict')
+    c.setopt(pycurl.HTTPHEADER, ['Accept:application/json', 'Content-Type: multipart/form-data'])
+    c.setopt(pycurl.HTTPPOST, [('image', (pycurl.FORM_FILE, "assets/stc.jpg"))])
+    c.setopt(pycurl.WRITEFUNCTION, b.write)
+    c.perform()
+    assert c.getinfo(pycurl.RESPONSE_CODE) == 200
+    c.close()
+
+    response = b.getvalue()
+    response = json.loads(response)
+
+    assert response['status'] == 'ok'
+    assert response['image_size'] == [513, 256]
+    assert len(response['seg_map']) == response['image_size'][1]
+
```

Comment:
Can you please add a sanity check here to ensure that the returned image segmentation data makes sense? No need to check the entire segmentation map; just verify that, say, three different points in the test image are still labeled with the same label they had when the model was known to be correct.

Reply:
Done